### PR TITLE
pounce: init at 1.0p1

### PR DESCRIPTION
**Pull request by edef1c**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/pounce/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    # Reference openssl(1) by absolute path
+    (fetchpatch {
+      url = https://code.causal.agency/june/pounce/commit/973f19b4fe73ef956fbb4eeaf963bbb83c926203.patch;
+      sha256 = "1w4rhwqfcakzb9a6afq788rrsypay0rw75bjk2f3l66spjb7v3ps";
+    })
+  ];
+
+  buildInputs = [ libressl ];
+
+  configurePhase = "ln -s Linux.mk config.mk";
```
Comment (alyssais) [inline pkgs/servers/pounce/default.nix]: Should probably only do this on Linux

File: pkgs/servers/pounce/default.nix [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{ stdenv, libressl, fetchzip, fetchpatch }:
+stdenv.mkDerivation rec {
```
Comment (alyssais) [inline pkgs/servers/pounce/default.nix]:
```suggestion

stdenv.mkDerivation rec {
```

File: pkgs/servers/pounce/default.nix (hunk 1 above)
Comment (edef1c) [reply pkgs/servers/pounce/default.nix]: `Linux.mk` is fairly innocuous, and `Darwin.mk` expects to find LibreSSL in `/usr/local/opt/libressl`. I'm not really sure what to do there yet, and can't really test.

Comment (doronbehar) [reply pkgs/servers/pounce/default.nix]: Then I'd suggest currently disable other platforms then Linux.

**Review by doronbehar — CHANGES_REQUESTED** [review]

File: pkgs/servers/pounce/default.nix (hunk 1 above)
Comment (jtojnar) [reply pkgs/servers/pounce/default.nix]: Also it should probably be in `preConfigure`.

**Pull request merged**